runner.py

```diff
@@ -1,4 +1,17 @@
 from app.game_manager import Game
 
-game = Game()
-game.show_welcome_screen()
+if "__main__" == __name__:
+    game = Game()
+    try:
+        with game.term.fullscreen():
+            # clear the second buffer screen
+            print(game.term.home + game.term.clear)
+
+        print(game.term.enter_fullscreen)
+        # Enter full screen once
+        game.start_game()
+    except KeyboardInterrupt:
+        # catch if user wants to exit by ctrl+c
+        # and exit full-screen
+
+        print(game.term.home + game.term.clear + game.term.exit_fullscreen)
```

Comment:
Rather than a `runner.py` move this to `__main__.py` so it would be ran when you do `python -m app`.

Reply:
Ahh yes

Reply:
Mentioned in ad98a0f.